File: Easy/TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        // declare an integer array
        int[] indices;

        // intialize array
        indices = new int[2]; 

        // loop through the given array
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                // if the numbers at the indices sum to the target, store the numbers into the array
                if (nums[i] + nums[j] == target) {
                    indices[0] = i;
                    indices[1] = j;
                    break;
                }
            }
        }
        // return the array
        return indices;
    }
}

File: Easy/TwoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """

        indexed_sum = [(nums[i], i) for i in range(len(nums))]
        indexed_sum.sort()

        low = 0
        high = len(nums) - 1

        while (low <= high):
            current_sum = indexed_sum[low][0] + indexed_sum[high][0]

            if (current_sum == target):
                return [indexed_sum[low][1], indexed_sum[high][1]]
            elif target > current_sum:
                low += 1
            else:
                high -= 1

        return -1  